src/NUnitFramework/framework/Internal/Filters/OrFilter.cs

```diff
@@ -111,20 +108,13 @@ public override bool Match( ITest test )
         /// <returns>True if any of the component filters explicit match, otherwise false</returns>
         public override bool IsExplicitMatch( ITest test )
         {
-            foreach( TestFilter filter in Filters )
-                if ( filter.IsExplicitMatch( test ) )
-                    return true;
-
-            return false;
+            return Filters.Any(filter => filter.IsExplicitMatch(test));
```

Comment:
Context: Some parts of the framework were updated recently to move away from linq to avoid closure allocations. I can't recall where, but based on the file history I don't believe it was the filters here so I think this should be ok.